examples_gbs/run_tutorial_vibronic.py

```diff
@@ -0,0 +1,123 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+.. _gbs-vibronic-tutorial:
+
+Vibronic Spectra Tutorial
+=========================
+In this tutorial, we study how GBS can be used to compute vibronic spectra. So let's start from
+the beginning: what is a vibronic spectrum? Molecules absorb light at frequencies that
+depend on the allowed energy transitions between different molecular configurations. These

```

Comment:
"between different states" or something similar will be better. Configuration is usually used to refer to different structures of molecules.